backend/src/controllers/link.controller.js

```
@@ -18,20 +18,20 @@ class LinkController {
       });
     }
-    const allLinks = await linkService.getLinksByHelperId(helperId);
-    if (order && order > allLinks.length + 1) {
-      return res.status(400).json({
-        errors: [{ msg: "Invalid value for order" }],
-      });
-    }
-
     try {
+      const allLinks = await linkService.getLinksByHelperId(helperId);
+      if (order && (isNaN(order) || order > allLinks.length + 1 || order < 0)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)